pkg/cmd/roachtest/tests/online_restore.go

```diff
@@ -242,6 +153,79 @@ func registerOnlineRestore(r registry.Registry) {
 	}
 }
 
+func registerOnlineRestoreCorrectness(r registry.Registry) {
+	sp := restoreSpecs{
+		hardware: makeHardwareSpecs(hardwareSpecs{workloadNode: true}),
+		backup: makeRestoringBackupSpecs(backupSpecs{
+			nonRevisionHistory: true,
+			version:            fixtureFromMasterVersion,
+			workload:           tpccRestore{opts: tpccRestoreOptions{warehouses: 10, workers: 1, waitFraction: 0}}}),
+		timeout:                15 * time.Minute,
+		suites:                 registry.Suites(registry.Nightly),
+		restoreUptoIncremental: 1,
+		namePrefix:             "online/correctness",
```

Comment:
what is the full name of the test after `roachtest` builds?

Reply:
For this example, it would be `restore/online/correctness/tpc-c/10/aws/inc-count=1/nodes=4/cpus=8/`

Reply:
nit: could you modify `tpccRestore.String()` to to return "tpcc" so its inline with tpce?